extensions/ql-vscode/src/view/resultsView.css

```diff
@@ -108,8 +119,12 @@ td.vscode-codeql__path-index-cell {
   text-align: right;
 }
 
-td.vscode-codeql__location-cell {
-  text-align: right;
+.vscode-codeql__alert-message-cell {
+  text-align: left !important;
```

Comment:
Why are these `!important`s necessary?

Reply:
I'll document them. They're needed to override the `th { text-align: center }` above in the file because of css specificity rules. Even though these clauses occur after the `th { ... }`, they don't override it.